packages/shell-api/src/cursor.ts

```diff
@@ -292,4 +292,11 @@ export default class Cursor extends ShellApiClass {
     throw new MongoshUnimplementedError(
       '`maxScan()` was removed because it was deprecated in MongoDB 4.0');
   }
+
+  @returnType('Cursor')
+  @serverVersions(['3.2.0', ServerVersions.latest])
```

Comment:
If you don’t mind me asking, are these two values minimum and maximum supported version?

Reply:
Definitely don't mind you asking! Those are the min/max supported *server* version, so it will still exist if the user is connected to a different server version but it won't be suggested by the autocomplete.